src/client/index.js

```diff
@@ -39,6 +39,12 @@ export function init ({ appContext, extraContext, pageViewContext }) {
 	//  Initialise click event tracking for interactive elements
 	oTracking.click.init('cta');
 
+	if (window.oTracking) {
+		console.warn("An oTracking instance already exists on window, skipping");
```

Comment:
**suggestion**: could we include the old and new instances to help with debugging?
```suggestion
		console.warn("An oTracking instance already exists on window, skipping", { currentInstance: window.oTracking, ourInstance: oTracking });
```